rooted-ota.sh

```diff
@@ -62,9 +62,33 @@ SKIP_MODULES=${SKIP_MODULES:-'false'}
 # Upload OTA to test folder on OTA server
 UPLOAD_TEST_OTA=${UPLOAD_TEST_OTA:-false}
 
-OTA_CHANNEL=${OTA_CHANNEL:-stable} # Alternative: 'alpha'
 NO_COLOR=${NO_COLOR:-''}
-OTA_BASE_URL="https://releases.grapheneos.org"
+OTA_VENDOR=${OTA_VENDOR:-graphene} # graphene, lineage, ...
+case $OTA_VENDOR in
+  graphene)
+    OTA_TYPE=${OTA_TYPE:-'ota_update'} # Other option: factory
+    OTA_BASE_URL=${OTA_BASE_URL:-"https://releases.grapheneos.org"}
+    OTA_CHANNEL=${OTA_CHANNEL:-stable} # Alternative: 'alpha'
+    ;;
+  lineage)
+    OTA_TYPE=${OTA_TYPE:-'nightly'} # Other option: microG (not really supported yet)
+    OTA_BASE_URL=${OTA_BASE_URL:-"https://download.lineageos.org/api/v2"}
+    if [ -n "$OTA_CHANNEL" ]; then
+      echo "Unsetting unused OTA_CHANNEL $OTA_CHANNEL for LineageOS"
```

Comment:
Those nice `print`, `printRed` and `printGreen` functions are not available yet. And I didn't want to move them before this just now.

Maybe they could be moved into a separate file within the upcoming modularization work! :slightly_smiling_face: